src/js/tosdr.js

```diff
@@ -90,10 +89,38 @@ $(function () {
         $('html, body').animate({
           scrollTop: $("#services").offset().top
         }, 1000);
-      }
+      } 
     } else {
       showModal(hash);
     }
   }
+  
+  if (location.search.indexOf('?') == 0) {
+    var GETarguments = location.search.substring(1)
+    var splits = GETarguments.split("=");
+    if (splits.length == 2 && splits[0] == "servicePoint" && splits[1].length > 0) {
```

Comment:
Cool, will do.  Also, I think I will change:  
- line 120 -
+   } else {  // also add GET parameter support for basic service referencing
+      showModal(GETarguments);
+    }
to have a specific condition:  
else if (has 'service' as GET parameter and a value of greater > 0 chars in length) { show service modal }
looking to the parameter itself for a value is not a good idea.

thanks -- will create another pull request.  :)